src/js/tech/html5.js

```diff
@@ -46,7 +46,7 @@ class Html5 extends Tech {
     // 1) Check if the source is new (if not, we want to keep the original so playback isn't interrupted)
     // 2) Check to see if the network state of the tag was failed at init, and if so, reset the source
     // anyway so the error gets fired.
-    if (source && (this.el_.currentSrc !== source.src || (options.tag && options.tag.initNetworkState_ === 3))) {
```

Comment:
I think removing the `initNetworkState_` check breaks `if not, we want to keep the original so playback isn't interrupted`